completions/man

```diff
@@ -56,16 +56,15 @@ _man()
         return
     fi
 
-    local manpath="$MANPATH"
-    [[ -z $manpath ]] && \
-        manpath=$( manpath 2>/dev/null || command man -w 2>/dev/null )
+    local manpath=$( manpath 2>/dev/null || command man -w 2>/dev/null )
     [[ -z $manpath ]] && manpath="/usr/share/man:/usr/local/share/man"
```

Comment:
I think we should continue to support systems for which neither `manpath` or `man -w` return useful info. I.e. just swap the order around; don't just drop use of `$MANPATH`.